awesomemotive/affwp-generator
# Issue 5 - Fix undefined campaign error when generating transactions

Campaign should be "campaigns". Simple fix.

## TESTING
1. Install and activate EDD. 
2. Save EDD as an integration (AffiliateWP >> Settings >> Integration tab)
3. In your wp-config file set `WP_DEBUG_LOG` to true.
4. Have this repo and the [WP CLI Generator](https://github.com/awesomemotive/affiliatewp-wp-cli-generator) installed and activated.
5. Run this command in terminal "wp affwp generate transactions edd --number=1" 
6. When you're on the master branch, confirm you get an error in the debug log like `PHP Notice:  Undefined index: campaign`, but when you switch to this branch and run the command again, there should be no more error.
